firebase-vscode/package.json

```diff
@@ -63,7 +63,8 @@
         },
         "firebase.firebasePath": {
           "type": "string",
-          "markdownDescription": "%ext.config.firebasePath%"
+          "markdownDescription": "%ext.config.firebasePath%",
+          "scope": "application"
```

Comment:
Upon a closer look, I think we want `machine-overridable`. To quote the **scope** section in the [schema](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema), emphasis mine: 

> `machine` - Machine specific settings that can be set only in user settings or only in remote settings. For example, **an installation path which shouldn't be shared across machines**.
> `machine-overridable` - Machine specific settings that can be overridden by workspace or folder settings.

The `-overridable` is mostly so that developers can pin to a different CLI / Node.js version for a certain workspace folder if they really want to. We won't programatically set this, it's all up to the developer